cfunits-python/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: cfunits-python
+  version: 1.1
+
+source:
+  git_url: https://bitbucket.org/cfpython/cfunits-python.git
+  git_tag: v1.1
+
+build:
+  number: 0
+  skip: True # [win and py3k]
+
+requirements:
+  build:
+    - python >=2.7,<3
+    - udunits2
+    - netcdf4
+  run:
+    - python >=2.7,<3
```

Comment:
Use that restriction only on `build` as I mentioned in my previous comment.

The reason for this is because conda has some weird behavior and removes The Python build string if you specify on both `build` and `run`.